### PR TITLE
Feature/board render

### PR DESCRIPTION
**Pull request by reneemes**
Updated the #render method on the cell class and made a #render method on the Board class. Woot woot!

### CODE REVIEW

File: lib/board.rb [hunk 1]
```diff
@@ -54,6 +56,20 @@ def place(ship, desired_coordinates)
             @cells[coordinate].place_ship(ship)
         end
     end
-end
 
+    def render(optional = nil)
```
Comment (TDManning) [inline lib/board.rb:60]: This is great Renee!

File: lib/cell.rb [hunk 2]
```diff
@@ -32,16 +29,19 @@ def fired_upon?
...
+    
+    # require 'pry'; binding.pry
+    if empty? && fired_upon?
       return "M"
     elsif !empty? && @ship.health == 0 && @ship.sunk? == true
       return "X"
     elsif fired_upon? && !empty?
-      return "H"   
-    elsif optional 
+      return "H"
```
Comment (TDManning) [inline lib/cell.rb:40]: All code is passing with these modifications so lets keep them!

**Review by TDManning — COMMENTED** [review]
All code is passing and this completes the iteration.

**Comment by CPPittman** [comment]
So glad to see that mystery "S" disappear

**Pull request merged**